src/components/invest/investslug/components/InvestComicFaq.tsx

```diff
@@ -0,0 +1,93 @@
+'use client'
+
+import { useState, useRef, useEffect } from 'react'
+import { PlusIcon, MinusIcon } from 'lucide-react'
+import { cn } from '@/lib/utils'
+
+interface FAQItemProps {
+  section: string
+  answer: string
+  image?: string
+}
+
+interface FAQSectionProps {
+  faqs: FAQItemProps[]
+}
+
+export const FaqSection: React.FC<FAQSectionProps> = ({ faqs }) => {
+  return (
+    <section className='w-full flex flex-col justify-center align-center '>
+      <div className='flex flex-col'>
+        {faqs.map((faq, index) => (
+          <FAQItem key={index} item={faq} isLast={index === faqs.length - 1} />
+        ))}
+      </div>
+    </section>
+  )
+}
+
+const FAQItem: React.FC<{ item: FAQItemProps; isLast?: boolean }> = ({ item, isLast }) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  const contentRef = useRef<HTMLDivElement>(null)
+  const [maxHeight, setMaxHeight] = useState('0px')
+  const [isCollapsing, setIsCollapsing] = useState(false)
+
+  useEffect(() => {
```

Comment:
hard to figure out why do we need `useEffect` here?

Think we can do this without it?
Also we should clear timeout on component unmount.

Basically store timeout id and do:
```
     return () => {
        clearTimeout(timeoutId)
      }
```

Reply:
will do after i solve responsiveness